frontend/packages/kubevirt-plugin/integration-tests-utils/enums/provisionSource.ts

```diff
@@ -10,13 +10,15 @@ export class ProvisionSource extends ObjectEnum<string> {
   static readonly CONTAINER = new ProvisionSource(
     'Container',
     'Import via Registry (creates PVC)',
-    'quay.io/kubevirt/fedora-cloud-container-disk-demo:latest',
+    'quay.io/kubevirt/cirros-container-disk-demo:latest',
```

Comment:
I changed the default container to cirros to make it smaller/faster

Reply:
We're using cirros long time ago and change it to fedora because some tests configured with cloud-init need to use fedora image, like guest-agent tests. 
How about let's have both of them and only use Fedora when necessary?